apps/condo/domains/ticket/access/TicketFile.js

```diff
@@ -0,0 +1,54 @@
+
+/**
+ * Generated by `createschema ticket.TicketFile 'organization:Text;file?:File;ticket?:Relationship:Ticket:SET_NULL;'`
+ */
+const get = require('lodash/get')
+const { getById } = require('@core/keystone/schema')
+const { checkOrganizationPermission } = require('@condo/domains/organization/utils/accessSchema')
+ 
+
+async function canReadTicketFiles ({ authentication: { item: user } }) {
+    if (!user) return false
+    if (user.isAdmin) {
+        return {}
+    }
+    return { organization: { employees_some: { user: { id: user.id }, isBlocked: false } } }
+}
+
+async function canManageTicketFiles ({ authentication: { item: user }, originalInput, operation, itemId }) {
+    if (!user) return false
+    if (user.isAdmin) return true
+    if (operation === 'create') {
+        const organizationIdFromTicketFile = get(originalInput, ['organization', 'connect', 'id'])
+        if (!organizationIdFromTicketFile) {
+            return false
+        }
+        const canManageTicketFiles = await checkOrganizationPermission(user.id, organizationIdFromTicketFile, 'canManageTickets')
+        return canManageTicketFiles
+    } else if (operation === 'update') {
+        const ticketFile = await getById('TicketFile', itemId)
+        if (!ticketFile) {
+            return false
+        }
+        const { ticket, createdBy, organization } = ticketFile
+        if (!ticket) {
+            // Temp file that wasn't connected to ticket
+            return (createdBy === user.id)
+        }
+        if (!organization) {
+            return false
+        }
+        const canManageTickets = await checkOrganizationPermission(user.id, organization, 'canManageTickets')
+        return canManageTickets
```

Comment:
Why just not `return await checkOrganizationPermission…`?

Reply:
Same, because it is a bad code style. Read ESlint no-return-await spec